index.html

```diff
@@ -386,7 +392,15 @@
       return pc.setLocalDescription(o, fix, fix);
     }, fix);
     return 0;
- }
+  },
+
+  untz : function(obj) {
+    untz = true;
+  },
+
+  nountz : function(obj) {
```

Comment:
Would be a little nicer to do this as `/untz start` and `/untz stop`, like with the sounds.